### ISSUE TITLE
Upgrade OpenJDK to 13

### ISSUE THREAD

**Issue by Sir4ur0n** [NixOS/nixpkgs#73330]
**Describe the bug**
The current [OpenJDK version](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/compilers/openjdk/default.nix) is 12 (GA) while the 13 (GA) was [released on the 2019-09-17](https://openjdk.java.net/projects/jdk/13/).

It would be great to provide the 13 version.

**Expected behavior**
The default is OpenJDK13, or at least, the version 13 is available somewhere (e.g. as package `openjdk13`).

**Additional context**
JDK 13 provides several useful changes, including an improved switch syntax, a better GC, and a [major fix on HttpClient](https://bugs.openjdk.java.net/browse/JDK-8221395).

**Metadata**
Please run `nix run nixpkgs.nix-info -c nix-info -m` and paste the result.
```
$ nix run nixpkgs.nix-info -c nix-info -m
nix: src/libexpr/attr-set.hh:54: void nix::Bindings::push_back(const nix::Attr&): Assertion `size_ < capacity_' failed.
[1]    22771 abort (core dumped)  nix run nixpkgs.nix-info -c nix-info -m
```

**Comment by prusnak** [comment]
This is not a bug but a package request.

**Comment by Sir4ur0n** [comment]
My bad, closed and opened #73335
